docs/api/animation/AnimationAction.html

```diff
@@ -11,121 +11,363 @@
+        to false, when the last loop of the action has finished, so that this action has no further 
+        impact.<br /><br />
+        
+        Default is false.<br /><br />
+        
+        Note: *clampWhenFinished* has no impact, if the action is interrupted (it has only an effect if 
+        its last loop has really finished).
+    </div>    
+
+    <h3>[property:Boolean enabled]</h3>
+    <div>
+        Setting *enabled* to *false* disables this action, so that it has no impact. Default is *true*.<br /><br />
+        
+        When the action is re-enabled, the animation continues from its current point of [page:.time time] 
+        (setting *enabled* to *false* doesn't reset the action).<br /><br />
+
+        Note: Setting *enabled* to *true* doesn’t automatically restart the animation. Setting *enabled* 
+        to *true* will only restart the animation immediately, if the following condition is fulfilled: 
+        [page:.paused paused] is *false*, this action has not been deactivated in the meantime (by 
+        executing a [page:.stop stop] or [page:.reset reset] command), and neither [page:.weight weight] 
+        nor [page:.timeScale timeScale] is 0.
+    </div>
+    
+    <h3>[property:Number loop]</h3>
+    <div>
+        The looping mode (can be changed with [page:.setLoop setLoop]). Default is 
+        [page:Animation THREE.LoopRepeat] (with an infinite number of [page:.repetitions repetitions])<br /><br />
+        
+        Must be one of these constants:<br /><br /> 
+        [page:Animation THREE.LoopOnce] - playing the clip once,<br /> 
+        [page:Animation THREE.LoopRepeat] - playing the clip with the choosen number of *repetitions*, 
+        each time jumping from the end of the clip directly to its beginning,<br /> 
+        [page:Animation THREE.LoopPingPong] - playing the clip with the choosen number of *repetitions*, 
+        alternately playing forward and backward.
+    </div>
+    
+    <h3>[property:Boolean paused]</h3>
+    <div>
+        Setting *paused* to *true* pauses the execution of the action by setting the effective time scale
+        to 0. Default is *false*.<br /><br />
+        
+        Setting *paused* to *true* doesn’t reset the action. The action can be continued from this point 
+        of time by setting *paused* to false afterwards. Unlike setting [page:.enabled enabled] to false
+        the current frame will be shown statically during pausing.<br /><br />
+
+        Note: Setting *paused* to *false* doesn’t automatically restart the animation. Setting *paused* to 
+        *false* will only restart the animation immediately, if the following condition is fulfilled:
+        [page:.enabled enabled] is *true*, this action has not been deactivated in the meantime (by executing 
+        a [page:.stop stop] or [page:.reset reset] command), and neither [page:.weight weight] nor 
+        [page:.timeScale timeScale] is 0.
```

Comment:
> Setting *paused* to *true* doesn’t reset the action. The action can be continued from this point 
 of time by setting *paused* to false afterwards. Unlike setting [page:.enabled enabled] to false
the current frame will be shown statically during pausing.<br /><br />
 
> Note: Setting *paused* to *false* doesn’t automatically restart the animation. Setting *paused* to 
*false* will only restart the animation immediately, if the following condition is fulfilled:
[page:.enabled enabled] is *true*, this action has not been deactivated in the meantime (by executing 
 a [page:.stop stop] or [page:.reset reset] command), and neither [page:.weight weight] nor 
 [page:.timeScale timeScale] is 0.

I think you are over-explaining things a bit here  - essentially `paused` works exactly as one would expect (it pauses on the current frame if set to true, and continues from that point if changed back to false). If you stop / reset the animation in the meantime, paused will not restart the animation, but that doesn't need explaining. 

The only additional information I would include is that the mechanism it uses to do so is setting the timescale to 0 (which you have included in the first sentence).